code/style.css

```diff
@@ -1,4 +1,189 @@
 html {
-  background: #0025ff;
+  background: black;
   color: #fff;
+  font-family: 'Times New Roman', Times, serif;
 }
+
+body {
+  margin: 0px;
+  padding: 0px;
+  width: 100%;
+}
+
+.header {
+  background: rgb(8, 16, 8);
+  width: 100%;
+  padding: 5px 0px;
+  margin: 0px 0px 0px 0px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header:hover {
+  font-size: 20px;
+  background-color: rgb(6, 26, 6);
+}
+
+.left {
+  display: flex;
+  justify-content: left;
+}
+
+.middle {
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.right {
+  display: none;
+  justify-content: right;
+}
+
+.title {
+  font-size: 30px;
+  justify-content: center;
+  letter-spacing: 2px;
+}
+
+.logo {
+  width: 50px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.navigation-bar {
+  display: flex;
+  align-items: center;
+}
+
+.navigation-items {
+  margin-right: 10px;
+}
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.big-picture {
+  width: 80%;
+  margin: 20px;
+  border: 5px rgb(16, 102, 102) double;
+  padding: 10px;
+}
+
+.big-newstext {
+  width: 80%;
+  background: rgb(8, 16, 8);
+  border: 5px rgb(16, 102, 102) double;
+  margin: 20px;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.big-newstext:hover {
+  background-color: rgb(6, 26, 6);
+}
+
+.big-newstext p {
+  font-size: 40px;
+  letter-spacing: 2px;
+}
+
+.small-cards {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center; 
+}
+
+.news-card2 {
+  width: 80%;
+  height: 320px;
+  background: whitesmoke;
+  color: #36806a;
+  border-radius: 10px;
+  margin: 20px 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.small-image {
+  width: 80%;
+  height: 200px;
+  margin-top: 20px;
+}
+
+.small-text {
+  margin: 10px;
+}
+
+@media (min-width: 768px) and (max-width: 992px) {
+  .left {
+    display: flex;
+    justify-content: center;
+  }
+
+  .small-cards {
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap;
+  }
+
+  .news-card2 {
+    width: 40%;
+  }
+
+  .small-image {
+    border-radius: 50%;
+  }
```

Comment:
If you want these to be completely round, they need to be completely square (before the border-radius is added, that is).